network/p2p/peerstore/peerstore.go

```diff
@@ -322,7 +328,7 @@ func (ps *PeerStore) appendTime(peerID peer.ID, t time.Time) {
 	_ = ps.Put(peerID, addressDataKey, ad)
 }
 
-// PopEarliestTime removes the earliest time from recentConnectionTimes in
+// popNElements removes the earliest time from recentConnectionTimes in
```

Comment:
Maybe `popEarliestN` ?

Reply:
it is the same as in `phonebook` so I'd preserve the name